src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1693,6 +1660,41 @@
       "required": [ "tool" ]
     },
 
+
+    "runAutomationDetails": {
+      "description": "Information that describes a run's identity and role within an engineering system process.",
+      "additionalProperties": false,
+      "type": "object",
+      "properties": {
+
+        "description": {
+          "description": "A description of the role played within the engineering system by this object's containing run object.",
```

Comment:
>  [](start = 46, length = 1)

Nit: The spec says here "identity and role", which I think is useful as it captures the purpose of `instanceGuid` and the trailing component of `instanceId`.